google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -2749,6 +2939,8 @@ private SessionPool(
     this.initOpenTelemetryMetricsCollection(openTelemetry, attributes);
     this.waitOnMinSessionsLatch =
         options.getMinSessions() > 0 ? new CountDownLatch(1) : new CountDownLatch(0);
+    this.multiplexedSessions = new ArrayDeque<>();
+    this.multiplexedSessionsInitialized = new CountDownLatch(1);
```

Comment:
nit: initialize these final fields where they are declared, as they do not depend on any of the input of this constructor

Reply:
Done